src/glibc.zig

```diff
@@ -649,6 +651,9 @@ fn add_include_dirs_arch(
             try args.append("-I");
             try args.append(try path.join(arena, &[_][]const u8{ dir, "riscv" }));
         }
+    } else if (arch.isLoongArch()) {
+        try args.append("-I");
+        try args.append(try path.join(arena, &[_][]const u8{ dir, "loongarch" }));
```

Comment:
Note that, after adding the extra headers I mentioned, you'll need to add the `nptl` subdirectory here like it's done for RISC-V above.

Reply:
done